sdk/servicebus/service-bus/src/core/streamingReceiver.ts

```diff
@@ -91,6 +91,11 @@ export class StreamingReceiver extends MessageReceiver {
    */
   protected _onAmqpMessage: OnAmqpEventAsPromise;
 
+  /**
+   * Whether we are currently registered for receiving messages.
+   */
+  public isReceivingMessages: boolean;
```

Comment:
From what I am seeing, setting `isReceivingMessages` to `true` is always accompanied by `helper.resume()` and setting it to `false` is always accompanied by `helper.suspend()`. Can we make this to a getter and that gets the value from the helper instead of setting/un-setting this boolean on the outside?

Reply:
I did a little renaming but got the gist of what you mentioned here and plumbed that through.